mods/species/bayliens/tajaran/datum/culture.dm

```diff
@@ -1,8 +1,13 @@
-/decl/cultural_info/culture/tajaran
-	name = "Tajaran Culture"
-	description = "The Tajaran culture is steeped in superstition and tradition from centuries of survival on their harsh, frozen homeworld."
+/decl/cultural_info/citizenship/tajaran
+	name = "Tajaran - Hrarar"
+	description = "Hrarar are the most numerous tajaran ethnicity on Meralar, and were the most heavily represented tajarans \
+	in the diplomatic delegations that dealt with humans through the early post-contact years. This led to a fairly entrenched \
+	image of the Hrarar as the \"average\" tajaran among humans and their other interstellar neighbours. Many Hrararan cultures \
+	are based on strong traditions and ideals, and the patchwork of nations and city-states that make up their lands often \
+	infight on matters of honour and religious one-upmanship."
```

Comment:
```suggestion
	squabble on matters of honour and religious one-upmanship."
```

Reply:
This whole thing is probably getting a rewrite when I do the species.